### ISSUE TITLE
Too slow, it takes 4 minutes to reason about one second of audio

### ISSUE THREAD

**Issue by c1a1o1** [MStypulkowski/diffused-heads#11]
Too slow, it takes 4 minutes to reason about one second of audio

**Comment by MStypulkowski** [comment]
Yes, the generation time is long. We didn't experiment with different samplers but feel free to try it.

@ngbien83 you used the model incorrectly. Please read the README section on how to use your own identity frames.

**Comment by liutaocode** [comment]
> r 4hrs rend

Does it disappear gradually?
